### ISSUE TITLE
Utilize Discord slash commands

### ISSUE THREAD

**Issue by jamesreiati** [jamesreiati/chill-bot#5]
https://discord.com/developers/docs/interactions/slash-commands

Currently waiting on underlying library support:
https://github.com/discord-net/Discord.Net/issues/1716

**Comment by amekudeku** [comment]
is it working right now?

**Comment by NotOfficer** [comment]
Whats the point of opening another issue?

**Comment by jamesreiati** [comment]
This issue describes the work for this project which would consume the APIs created by the underlying library. This issue does not describe the work for Discord.Net to implement slash commands. I didn't realize it would create a notification for the Discord.Net repository. I linked to the issue so that I could periodically check the progress.
